sympy/physics/mechanics/_system.py

```diff
@@ -656,6 +671,22 @@ def apply_gravity(self, acceleration):
         """
         self.add_loads(*gravity(acceleration, *self.bodies))
 
+    @_reset_eom_method
+    def add_actuators(self, *actuators):
+        """Add actuator(s) to the system.
+
+        Parameters
+        ==========
+
+        *actuators : subclass of ActuatorBase
+            One or more actuators.
+
+        """
+        self._check_objects(actuators, self.actuators, ActuatorBase,
+                            'Actuators',
```

Comment:
Formatting? And date I ask about why this method needs two strings of the same thing in different capitalisation cases?

Reply:
Sometimes they are actually more different, namely in the case of constraints. There the object name (e.g. holonomic constraints) and the type name (e.g. expressions) is not the same.

Reply:
Thanks.